h2o-docs/src/product/getting-data-into-h2o.rst

```diff
@@ -190,52 +192,135 @@ core-site.xml must be configured for at least the following properties (class, p
     </configuration>
 
 
-Direct Hive import
+Direct Hive Import
 ~~~~~~~~~~~~~~~~~~
 
-H2O supports direct ingestion of data managed by Hive in Hadoop. This feature is available only when H2O is running as a Hadoop job. Internally H2O uses metadata in Hive Metastore database to determine the location and format of given Hive table. H2O then imports data directly from HDFS so limitations of supported formats mentioned above apply. Data from hive can pulled into H2O using ``import_hive_table`` function. H2O can read Hive table metadata two ways - either via direct Metastore access or via JDBC.
+H2O supports direct ingestion of data managed by Hive in Hadoop. This feature is available only when H2O is running as a Hadoop job. Internally H2O uses metadata in Hive Metastore database to determine the location and format of given Hive table. H2O then imports data directly from HDFS so limitations of supported formats mentioned above apply. Data from hive can pulled into H2O using ``import_hive_table`` function. H2O can read Hive table metadata two ways - either via direct Metastore access (preferred) or via JDBC. 
```

Comment:
actually I dont think either way of reading hive metadata is preffered, they both work well

maybe you have this mistaken with the Direct hive import being prefered over reading data from Hive via JDBC

Reply:
and the JDBC access to metadata is actually easier to setup